quarkus/service/src/main/resources/application.properties

```diff
@@ -120,30 +120,18 @@ polaris.authentication.token-broker.max-token-generation=PT1H
 # polaris.storage.gcp.token=token
 # polaris.storage.gcp.lifespan=PT1H
 
+# Profile for unit tests
 %test.quarkus.log.file.enable=false
-%test.quarkus.log.category."org.apache.polaris".level=INFO
-%test.quarkus.log.category."org.apache.iceberg.rest".level=INFO
-%test.quarkus.log.category."org.apache.iceberg.rest.RESTSessionCatalog".level=ERROR
-%test.quarkus.log.category."org.apache.polaris.core.persistence.PolarisMetaStoreManagerImpl".level=ERROR
-%test.quarkus.log.category."org.apache.polaris.service.context.DefaultRealmContextResolver".level=ERROR
-%test.quarkus.log.category."org.apache.polaris.service.catalog.PolarisCatalogHandlerWrapper".level=ERROR
-%test.quarkus.log.category."org.apache.polaris.service.storage.PolarisStorageIntegrationProviderImpl".level=ERROR
-%test.quarkus.http.limits.max-body-size=1000000
 %test.quarkus.otel.sdk.disabled=true
-%test.polaris.authentication.token-broker.type=symmetric-key
-%test.polaris.authentication.token-broker.symmetric-key.secret=polaris
-%test.polaris.features.defaults."ALLOW_EXTERNAL_CATALOG_CREDENTIAL_VENDING"=false
-%test.polaris.features.defaults."ALLOW_EXTERNAL_METADATA_FILE_LOCATION"=false
-%test.polaris.features.defaults."ALLOW_OVERLAPPING_CATALOG_URLS"=true
-%test.polaris.features.defaults."ALLOW_SPECIFYING_FILE_IO_IMPL"=true
-%test.polaris.features.defaults."ALLOW_WILDCARD_LOCATION"=true
-%test.polaris.features.defaults."ENFORCE_PRINCIPAL_CREDENTIAL_ROTATION_REQUIRED_CHECKING"=true
-%test.polaris.features.defaults."INITIALIZE_DEFAULT_CATALOG_FILEIO_FOR_TEST"=true
-%test.polaris.features.defaults."SKIP_CREDENTIAL_SUBSCOPING_INDIRECTION"=true
-%test.polaris.features.defaults."SUPPORTED_CATALOG_STORAGE_TYPES"=["FILE","S3","GCS","AZURE"]
-%test.polaris.realm-context.realms=POLARIS
-%test.polaris.realm-context.type=test
-%test.polaris.storage.aws.access-key=accessKey
-%test.polaris.storage.aws.secret-key=secretKey
-%test.polaris.storage.gcp.token=token
-%test.polaris.storage.gcp.lifespan=PT1H
+
+# Profile for integration tests
+%it.quarkus.http.limits.max-body-size=1000000
+%it.polaris.features.defaults."ALLOW_EXTERNAL_CATALOG_CREDENTIAL_VENDING"=false
+%it.polaris.features.defaults."ALLOW_EXTERNAL_METADATA_FILE_LOCATION"=false
+%it.polaris.features.defaults."ALLOW_OVERLAPPING_CATALOG_URLS"=true
+%it.polaris.features.defaults."ALLOW_SPECIFYING_FILE_IO_IMPL"=true
+%it.polaris.features.defaults."ALLOW_WILDCARD_LOCATION"=true
+%it.polaris.features.defaults."ENFORCE_PRINCIPAL_CREDENTIAL_ROTATION_REQUIRED_CHECKING"=true
+%it.polaris.features.defaults."INITIALIZE_DEFAULT_CATALOG_FILEIO_FOR_it"=true
+%it.polaris.features.defaults."SKIP_CREDENTIAL_SUBSCOPING_INDIRECTION"=true
+%it.polaris.features.defaults."SUPPORTED_CATALOG_STORAGE_TYPES"=["FILE","S3","GCS","AZURE"]
```

Comment:
Would it be possible to keep these settings in a java test profile? IIRC, `application.properties` is embedded in the final distribution jar, so it would be nice to avoid having test stuff there.

Reply:
I mean using `QuarkusTestProfile` to define these properties... Does it not work with integration tests?

Reply:
No, you can't use `QuarkusTestProfile` with `@QuarkusIntegrationTests`.

Reply:
TBH I'm not a fan of splitting the files, I think having all the profiles in `application.properties` is actually simpler, wdyt?

Reply:
fair enough! TIL about test profiles not working in integration tests :thinking: